backend/handler/database/users_handler.py

```diff
@@ -42,4 +43,10 @@ def delete_user(self, id: int, session: Session = None):
 
     @begin_session
     def get_admin_users(self, session: Session = None) -> list[User]:
-        return session.scalars(select(User).filter_by(role=Role.ADMIN)).all()
+        try:
+            return session.scalars(select(User).filter_by(role=Role.ADMIN)).all()
+        except ProgrammingError as e:
+            if "Table" in str(e.orig) and "doesn't exist" in str(e.orig):
+                return []
+            else:
+                raise ProgrammingError from e
```

Comment:
Can you add a comment explaining this error block?

Reply:
hmm i can't remember why i added that there, maybe something in the middle of the development, i'll remove it